src/components/FamilyManifestModal/FamilyManifestModal.js

```diff
@@ -127,6 +128,38 @@ export default compose(
       }).then(x => this.setState(x));
     },
   }),
+  withFormik({
```

Comment:
this seems odd, there's already a similar form down below in the body doing a similar thing w/ `fileManifestParticipantsAndFamily`, but as far as i can tell this one wouldn't be used. is this tested?

Reply:
this looks like the old code, which was removed during the reimplementation of the modal... probably an artifact of merge conflict resolution...